python/makeALPACAEvents.py

```diff
@@ -120,25 +130,82 @@ def ntupleWrite(ctau,fn,Lmin,Lmax,startZ,endZ,SmearBeam,mres,gax,old):
         ntup.Fill(int(i),int(2),int(dau2[1]),float(dau2[7]),float(dau2[8]),float(dau2[9]),daux,dauy,dauz,int(0),float(1),float(dau2[10]),float(dau2[15])/10./c_light,w)
     ntup.Write()
     f.Close()
+    print("ALP -> Gamma + Gamma events are produced with cross-section of {} pb at mass of {} GeV and photon coupling of {} GeV^-1".format(xs,mres,gax))
     return inputFile
 
 def runEvents(mres,gax,nev,Lmin,Lmax,startZ,endZ,SmearBeam):
+    mres = float(mres)
+    gax  = float(gax)
     print('ALPACA is starting for mass of {} GeV with photon coupling coeffiecient {} GeV^-1.'.format(mres,gax))
-    pdg = ROOT.TDatabasePDG.Instance()
-    pdg.AddParticle('a','ALP', mres, False, gax, 0., 'a', 9900015)
+    addALPtoPDG(float(mres),float(gax))
```

Comment:
Casting again here seems superfluous.

I would suggest to rename the function arguments or local variables to make this less confusing.